core/src/core/pmmr/readonly_pmmr.rs

```diff
@@ -73,7 +73,9 @@ where
 			Some(p) => p,
 			None => self.size,
 		};
-		let mut pmmr_index = pmmr_index1 - 1;
+		// default index to 0 if arg index1 is 0
+		let mut pmmr_index = if pmmr_index1 > 1 { pmmr_index1 - 1 } else { 0 };
```

Comment:
The endless saga of 1 vs 0 indexed pmmrs.

This is fine as a belt+suspenders check, though I think the idea was that this would always be called assuming a 1 index, as exposed by all of the APIs. A lot of work was done to use 0 indices internally, but this confusion still abounds.

In any case, all fine, but you can just use `pmmr_index = pmmr_index1.saturating_sub(1);' instead of if statements